spec/spec_helper.rb

```diff
@@ -0,0 +1,52 @@
+require 'ostruct'
+require 'factory_bot'
+require 'faker'
+require 'rspec'
+require 'simplecov'
+require 'webmock'
+
+SimpleCov.minimum_coverage 100
+WebMock.enable!
```

Comment:
Are we actually using webmock anywhere (or intend to in the future)?  It looked like we were just turning it off everywhere at a glance.

Reply:
We should only be turning it off for the integration tests:
```joe.ebmeier@sbf.dev:~$ find manifestly-ruby/spec/ -type f | xargs grep -i webmock
manifestly-ruby/spec/integration/user_spec.rb:    WebMock.allow_net_connect!
manifestly-ruby/spec/integration/user_spec.rb:    WebMock.enable!
manifestly-ruby/spec/integration/checklist_run_spec.rb:    WebMock.allow_net_connect!
manifestly-ruby/spec/integration/checklist_run_spec.rb:    WebMock.enable!
manifestly-ruby/spec/integration/workflow_spec.rb:    WebMock.allow_net_connect!
manifestly-ruby/spec/integration/workflow_spec.rb:    WebMock.enable!
manifestly-ruby/spec/spec_helper.rb:require 'webmock'
manifestly-ruby/spec/spec_helper.rb:WebMock.enable!```

Reply:
I think its fine since we are using it for the unit tests.